oval_graph/oval_node.py

```diff
@@ -22,6 +22,7 @@ class OvalNode():
         input_value (str): value of node
         input_negation (bool): value indicating whether the node is negated or not
         comment (str): text about node
+        tag (str): info about node if is test node, definition node or rule node
```

Comment:
You probably mean that tag specifies if the node represents OVAL test, OVAL definition or XCCDF rule.

That leads me to this question: Would it be better to use some enumeration for this instead of arbitrary string?

Reply:
Enumeration sounds good. I started think about implementation enumeration for values and operators too. I will implement enumeration in following PR.